.github/workflows/test.yml

```
@@ -40,6 +40,10 @@ jobs:
       - name: Build
         run: echo "Building on ${{ matrix.os }}"
 
+      - name: Add GNU tar to PATH (significantly faster than windows tar)
+        if: matrix.flavor.target == 'windows'
```

actionlint (1.7.4)

property "flavor" is not defined in object type {os: string; target: string}

(expression)